List of Wallets that Support ETC (#1280)
ethereumclassic/ethereumclassic.github.io: content/blog/2023-07-12-list-of-wallets-that-support-ethereum-classic/index.md

---
title: "List of Wallets that Support Ethereum Classic"
date: 2023-07-12
author: Donald McIntyre
contributors: ["DonaldMcIntyre"]
tags: ["guide", "series"]
linkImage: ./banner.png
---

Ethereum Classic (ETC) is a widely supported cryptocurrency in the blockchain industry. The majority of top wallets, exchanges, and mining pools offer ETC as one of their products, pairs, or mining alternatives.

We have covered the major wallet brands, and we will produce series about mining pools and exchanges as well.

> In this post, we list the six major wallets that we described in our "Using Ethereum Classic With" series. We do not recommend any over others as it depends on users to select the model and style that best fit their needs.

Reviewer: Line 14: Revise for clarity and verb agreement
```suggestion
In this post, we list the six major wallets we have described in our "Using Ethereum Classic With" series. We do not recommend any wallet over any other as it is up to users to select the model and style that best fits their individual needs.
```

The features and use cases for each brand are very distinct, so it is worth exploring all these alternatives before making a choice. All are non-custodial wallets, though, as the whole purpose of the blockchain industry is trust minimization!

Following we list the six posts and corresponding videos that we have written and produced with their summaries and links. There will be more!

## Using Ethereum Classic With MetaMask

> MetaMask the the most massively used wallet in the Ethereum EVM blockchain segment with more than 25 million monthly active users during the 2021 peak. It was the first browser extension for web3 use and then it added mobile support with their popular MetaMask for iOS and Android apps. To add ETC to MetaMask it is necessary to go through some manual steps that are described in the post. It is good for holding ETC, other ETC ERC-20 tokens, and trading and holding NFTs on ETC.

Reviewer: Line 22: Remove duplicate word; minor wording change
```suggestion
MetaMask is the most massively used wallet in the Ethereum EVM blockchain segment with more than 25 million monthly active users during the 2021 peak. It was the first browser extension for web3 use and then it added mobile support with their popular MetaMask for iOS and Android apps. To add ETC to MetaMask it is necessary to go through some manual steps described in the post. It is suitable for holding ETC, other ETC ERC-20 tokens, and trading and holding NFTs on ETC.
```

Video: 

<iframe width="560" height="315" src="https://www.youtube.com/embed/85xAV8jzpQQ" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>

## Using Ethereum Classic With Trust Wallet

Trust Wallet is widely used as well and was exclusively an Android and iOS mobile wallet, but now they have a browser extension as well. It hosts multiple crypto assets, including Bitcoin, Ethereum, Ethereum Classic, Litecoin, Zcash, and many more both Ethereum EVM compatible and others. As it already supports ETC by default, it is easy for users to activate it and start receiving and sending ETC.  

Link to post: https://ethereumclassic.org/blog/2023-02-07-using-ethereum-classic-with-trust-wallet

Video: 

<iframe width="560" height="315" src="https://www.youtube.com/embed/8rI_ADTDcGw" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe> 

## Using Ethereum Classic With Ledger Nano S

> Ledger Nano S is one of the three models of the Ledger family of hardware wallets. It is being upgraded as we write this post with Ledger Nano S Plus, so we will write a new post about the new upgrade, However, the installed base of Ledger Nano S users in enormous so this post will have a long shelf life. We will also cover the other two models of the Ledger family. Ledger nano S or Plus is a good alternative for users with larger amounts of ETC who want a higher degree of security than the classical mobile, browser extension, and desktop alternatives.

Reviewer: Line 42: Correct puncrtuation
```suggestion
Ledger Nano S is one of the three models of the Ledger family of hardware wallets. It is being upgraded as we write this post with Ledger Nano S Plus, so we will write a new post about the new upgrade. However, the installed base of Ledger Nano S users is enormous so this post will have a long shelf life. We also cover the other two models of the Ledger family. Ledger Nano S or Plus is a good alternative for users with larger amounts of ETC who want a higher degree of security than the classical mobile, browser extension, and desktop alternatives.
```

Video: 

<iframe width="560" height="315" src="https://www.youtube.com/embed/o4n6pqRyA1c" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>

## Using Ethereum Classic With MyEtherWallet

MyEtherWallet (MEW) features are very much the same as MyCrypto. Indeed, MEW was one of the first wallets in the Ethereum ecosystem, thus ETC, when it launched in 2015, and one of the co-founders left the startup to start MyCrypto! MEW is also a massively used wallet app, is web based, but also has mobile, browser extension, and connections to other wallets, including hardware wallets. MEW also supports ETC by default. 

Link to post: https://ethereumclassic.org/blog/2023-05-23-using-ethereum-classic-with-myetherwallet

Video: 

<iframe width="560" height="315" src="https://www.youtube.com/embed/Uv0oOHj23Gs" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>

## Using Ethereum Classic With Exodus

> Exodus was born as a desktop wallet and now has desktop and mobile apps that support Mac, Windows, iOS, and android. It also launched a browser extension to be present in the web3 revolution. Its focus is on esthetics and to be the best design and user experience. It natively supports ETC and it is very easy to activate. Exodus may be used with many other Ethereum and non Ethereum EVM blockchains as well. One of its focuses is security, so it features easy connections to hardware wallets so users may keep their private keys off their day to day devices.

Reviewer: Line 72: Capitalization
```suggestion
Exodus was born as a desktop wallet and now has desktop and mobile apps that support Mac, Windows, iOS, and Android. It also launched a browser extension to be present in the web3 revolution. Its focus is on aesthetics and to be the best design and user experience. It natively supports ETC and it is very easy to activate. Exodus may be used with many other Ethereum and non-Ethereum EVM blockchains as well.  For added security, it features easy connections to hardware wallets so users may keep their private keys off their day-to-day devices.
```
